corehq/apps/export/models/new.py

```diff
@@ -211,6 +231,51 @@ def generate_instance_from_schema(schema, domain, app_id=None):
         return instance
 
 
+class ExportInstanceDefaults(object):
+    """
+    This class is responsible for generating defaults for various aspects of the export instance
+    """
+    @staticmethod
+    def get_default_instance_name(schema):
+        raise NotImplementedError()
+
+    @staticmethod
+    def get_default_table_name(table_path):
+        raise NotImplementedError()
+
+    @staticmethod
+    def get_default_table_selected(path):
```

Comment:
hmm that almost sounds like is the default table selected when in fact we're getting the default selection for any table. what about `default_is_table_selected`, best solution is probably add a comment haha